runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -23,6 +23,9 @@
 import io.netty.util.concurrent.DefaultEventExecutorGroup;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'io.netty.util.concurrent.DefaultEventExecutorGroup'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)